src/main/java/com/welovecoding/netbeans/plugin/editorconfig/editor/csl/EditorConfigSectionStructureItem.java

```diff
@@ -0,0 +1,90 @@
+package com.welovecoding.netbeans.plugin.editorconfig.editor.csl;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+import javax.swing.ImageIcon;
+import org.netbeans.api.annotations.common.StaticResource;
+import org.netbeans.modules.csl.api.ElementHandle;
+import org.netbeans.modules.csl.api.ElementKind;
+import org.netbeans.modules.csl.api.HtmlFormatter;
+import org.netbeans.modules.csl.api.Modifier;
+import org.netbeans.modules.csl.api.StructureItem;
+import org.openide.util.ImageUtilities;
+
+/**
+ *
+ * @author junichi11
+ */
+public class EditorConfigSectionStructureItem implements StructureItem {
+
+  private final String section;
+  private final int startPosition;
+  private final int endPosition;
+  private final List<StructureItem> properties;
+  @StaticResource
+  private static final String ICON_PATH = "com/welovecoding/nbeditorconfig/section16.png"; // NOI18N
```

Comment:
Let's store the `ICON_PATH` in `com.welovecoding.netbeans.plugin.editorconfig.config.Settings.java` as we already have the `LOGO_PATH` there.